dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestForeignKey.java

```diff
@@ -74,12 +74,15 @@
 	public static void doTestSuiteSetup() throws Exception {
 		try{
 			_conn = DriverManager.getConnection(Utils.url, Utils.usr, Utils.pwd);
-			Statement stmt = _conn.createStatement();
-			
-			stmt.execute(strCreatePKTABLE1Query);
-			stmt.execute(strCreatePKTABLE2Query);
-			stmt.execute(strCreateFKTABLE1Query);
-			stmt.execute(strCreateFKTABLE2Query);
+            try (

```

Comment:
Thanks for the change Weiqing. Learnt something new related to try-with-resource-statement from your change - it does reduce the earlier code change planned - thanks.

Could you please also take care of replacing DriverMagager.getConnection calls from *Blob*, *Clob*, TestGetIndexInfo and this file ? This would take care random non-existent schema errors seen.

Reply:
I see you did a new commit to fix some review comments. Can you also fix this one ? This fix is needed for the daily builds to pass. We often get "non existent schema " errrors with JDBC tests.